data/actions/scripts/other/largeseashell.lua

```diff
@@ -3,7 +3,7 @@ function onUse(player, item, fromPosition, target, toPosition, isHotkey)
 		local chance = math.random(100)
 		local msg = ""
 		if chance <= 16 then
-			player:addHealth(-200)
```

Comment:
I dont think you should touch that one, its whole different method, so dont change it

Reply:
actually the original script used doTargetCombat, I was the one who changed it to addHealth because I was noob 😝

Reply:
why is there -100 to -200 if the code before had flat -200?

Reply:
because the damage is not flat, it is random and take armor into account, just like the hand of the noxius spawn

Reply:
https://tibia.fandom.com/wiki/Large_Seashell_(Closed)
so you wrong, min and max should be -200 not min -100 and max -200

Reply:
i hope its right now